docs/calamari/node-operators/join-network.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 3
+---
+
+# Join the network
+
+Specifically our test-net ----> https://telemetry.manta.systems/#/Calamari-Parachain%20Development
+
+## As a validator
+
+#### specific to this node
+  - ``my_node_name=my-awesome-manta-node``
+  - ``my_node_port=30333``
+  - ``my_node_ws_port=9944``
+  - ``my_db_path=/path/to/my/db``
+
+#### common for all nodes, you should have those files from the previous section
+  - ``relay_chain_spec_file=/full/path/to/rococo.json``
+  - ``para_chain_binary=/full/path/to/calamari-pc``
+
+```bash
+
+${para_chain_binary} \
+  --validator \
+  --name ${my_node_name} \
+  --base-path ${my_db_path} \
+  --chain calamari-dev \
+  --parachain-id 7777 \
+  --port ${my_node_port} \
+  --ws-port ${my_node_ws_port} \
+  --rpc-cors all \
+  --telemetry-url 'wss://api.telemetry.manta.systems/submit/ 0' \
+  --bootnodes /dns/alice.calamari.dev.manta.systems/tcp/30333/p2p/12D3KooWHPLqWuSDzVgMLor9DVcwoSxV27pKVqzuFcEH2ogyLNBu \
+  -- \
+  --bootnodes /dns/alice.rococo.mock.manta.systems/tcp/30333/p2p/12D3KooWJMcEQUbxq2CE1qoCqHCWxqjBfnpfeBCyqqrHBABJKbVr \
+  --chain ${relay_chain_spec_file}
+```
+  If it does not launch, give it extra permissions with:
+  
+  ``sudo chmod 777 ${para_chain_binary}``
```

Comment:
this ought to read: `sudo chmod +x ${para_chain_binary}` (or possibly `sudo chmod a+x ${para_chain_binary}`). there is no reason to grant 777 (world read, write, execute) on the binary and having this sort of thing in instructions will bite a copy-paster badly somewhere sometime.

Reply:
Ok, let's go with ``sudo chmod +x ${para_chain_binary}`` I've encountered it more often in instructions.